solr/solr-ref-guide/modules/indexing-guide/pages/field-type-definitions-and-properties.adoc

```diff
@@ -201,7 +201,7 @@ The table below includes the default value for most `FieldType` implementations
 |`sortMissingFirst`, `sortMissingLast` |Control the placement of documents when a sort field is not present. |`false`
 |`multiValued` |If `true`, indicates that a single document might contain multiple values for this field type. |`false`
 |`uninvertible` |If `true`, indicates that an `indexed="true" docValues="false"` field can be "un-inverted" at query time to build up large in memory data structure to serve in place of xref:docvalues.adoc[]. *Defaults to `true` for historical reasons, but users are strongly encouraged to set this to `false` for stability and use `docValues="true"` as needed.* |`true`
-|`omitNorms` |If `true`, omits the norms associated with this field (this disables length normalization for the field, and saves some memory). *Defaults to true for all primitive (non-analyzed) field types, such as int, float, data, bool, and string.* Only full-text fields or fields need norms. |*
+|`omitNorms` |If `true`, omits the norms associated with this field (this disables length normalization for the field, and saves some memory). *Defaults to true for all primitive (non-analyzed) field types, such as int, float, data, bool, and string.* Only full-text fields or ??? fields need norms. |*
```

Comment:
It seemed to me that a word was missing here, but I'm not sure what that word needs to be - appreciate the help here.

Reply:
I wish I knew what went here too..  if nothing comes ot mind, lets just edit it to `full-text fields need norms.` ;-).

Reply:
I believe I found the original phrase in other places in the codebase - e.g., in the techproducts schema https://github.com/apache/solr/blob/f0fcd300c896b858ae83235ecdb0a109eaea5cea/solr/server/solr/configsets/sample_techproducts_configs/conf/managed-schema.xml#L86-L87